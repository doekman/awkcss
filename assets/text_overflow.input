012345678901234567
Dit is een test.
Abc def ghijk lmn
Testenderwijs bla.
Nog meer tekst is dit.
-Gaat dit goedkomen?
+Jajaja, emojis zijn de toekomst.
*Een scene eindit met een cut als deze klaar is, of als het een verspilling van de film is.
