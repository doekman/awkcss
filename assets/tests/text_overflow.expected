012345678901234567
Dit is een test.  
Abc def ghijk lmn 
Testenderwijs bla.
Nog meer tekst is 
-Gaat dit goedkomâ€¦
+Jajaja, emojis 8<
*Een scene eindiðŸŽ¬
